llvm/include/llvm/Analysis/AltMathLibFuncs.def

```diff
@@ -76,7 +76,1098 @@ TLI_DEFINE_ALTMATHFUNC(Intrinsic::fpbuiltin_cos, Type::DoubleTyID, FIXED(2), "__
 
 #endif
 
-
+#if defined(TLI_DEFINE_SVML_ALTMATHFUNCS)
+//-----------------------SIN-----------------------//
+TLI_DEFINE_ALTMATHFUNC(Intrinsic::fpbuiltin_sin, Type::HalfTyID, FIXED(1), "__svml_sins1_ep", 32)
```

Comment:
Transplant test of these variants?

Reply:
@FreddyLeaf Can you please explain what you mean by this? Thanks.